### PR TITLE
Update STYLE_GUIDE.md

### PR DESCRIPTION
**Pull request by schettn**
Update rules for header comments and JSDoc header comments

### CODE REVIEW

File: STYLE_GUIDE.md [hunk 1]
```diff
@@ -63,6 +63,20 @@ let test = "test";
...
+/**
+ * This is a multiline header comment which contains single line JSDoc tags.
+ * @description And that's why you should never add add dots here
+ * @description Or here
+ */
+ 
+ /**
+  * This is a multiline header comment which contains multiline JSDoc tags.
+  * @description And that's why you should 
+  *              add add dots here.
```
Comment (schettn) [inline STYLE_GUIDE.md]: Hey Nico. You should remove one "add"

File: STYLE_GUIDE.md [hunk 2]
```diff
@@ -63,6 +63,20 @@ let test = "test";
 
 // This is a single line comment preffered to use in Javascript. 
 // It has multiple sentences.
+
+/**
+ * This is a multiline header comment which contains single line JSDoc tags.
+ * @description And that's why you should never add add dots here
```
Comment (pinterid) [inline STYLE_GUIDE.md]: 1 add to much

File: STYLE_GUIDE.md (hunk 1 above)
Comment (pinterid) [inline STYLE_GUIDE.md]: 1 add to much

**Review by pinterid — CHANGES_REQUESTED** [review]
Looks good after a few changes.

**Review by pinterid — APPROVED** [review]
Looks fine

**Pull request merged**